server/tailsql/static/script.js

```diff
@@ -20,6 +21,15 @@
         }
     })
 
+    body.addEventListener("keydown", (evt) => {
```

Comment:
Nit - `keyup` tends to be preferred for keybindings, the act of release the key is considered a marginally more reliable indicator of user intent, for some definitions of the word reliable.

Reply:
Done! I tried using keyup for the other shortcut, but it didn't bubble correctly, so I settled on keydown. But here it works fine 🤷🏼‍♀️